docs/uma2-api-descriptor.yml

```diff
@@ -148,6 +148,117 @@ paths:
           $ref: "#/components/responses/Unauthorized"
         403:
           $ref: "#/components/responses/Forbidden"
+  #Protection API - Access Policies
+  "/uma/protection/resource_set/{resource_id}/policies":
+    get:
+      tags:
+        - Protection API - Resource access policies
+      summary: List resource's access policies
+      parameters:
+        - in: path
+          name: resource_id
+          description: resource identifier
+          required: true
+          schema:
+            type: string
+          example: "9ca2a1d0-a310-4011-a0d2-eed7df99e1b6"
+      responses:
+        200:
+          description: Ok
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/AccessPolicy"
+        401:
+          $ref: "#/components/responses/Unauthorized"
+        403:
+          $ref: "#/components/responses/Forbidden"
+    post:
+      tags:
+        - Protection API - Resource access policies
+      summary: Create access policy
+      parameters:
+        - in: path
+          name: resource_id
+          description: resource identifier
+          required: true
+          schema:
+            type: string
+          example: "9ca2a1d0-a310-4011-a0d2-eed7df99e1b6"
+      requestBody:
+        $ref: "#/components/requestBodies/AccessPolicy"
+      responses:
+        201:
+          description: Created
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/AccessPolicy"
+        401:
+          $ref: "#/components/responses/Unauthorized"
+        403:
+          $ref: "#/components/responses/Forbidden"
+  "/uma/protection/resource_set/{resource_id}/policies/{policy_id}":
+    put:
```

Comment:
You missed the GET /uma/protection/resource_set/{resource_id}/policies/{policy_id}